### ISSUE TITLE
New Feature: Visão por Vereador

### ISSUE THREAD

**Issue by augustoqm** [antonioricardojr/vereador#14]
Tela Dinâmica com Visão por Candidato (candidatos a reeleição deve ser o default, e desmarcando algo mostraria todos)
- [ ] Qual o Partido?
- [ ] Eleito ou Suplente? 
- [ ] Se Suplente entrou quando? Se Eleito se afastou em algum momento?
- [ ] Quais seus principais temas?
- [ ] Qual sua eficiência temporal? (Mostrar gráfico com Quantidades de Ementas por Mês)
- [ ] Mudou de partido? (2008 - 2012 - 2016?) 
- [ ] Qual o Impacto Financeiro da nova Lei sobre ele? (Comparar gastos em 2008/2012 com o limite atual, ver site do [TSE com limites por municípios](http://www.tse.jus.br/eleicoes/eleicoes-2016/prestacao-de-contas/divulgacao-dos-limites-legais-de-campanha))

**Comment by augustoqm** [comment]
Quem bora? =D

**Comment by nazareno** [comment]
Massa! 
Mas repara que tirando o último checkbox, eu acho que os demais dados podem ser os mesmos que usaremos para o perfil dos vereadores no geral. Podem inclusive vir da API para as mesmas visualizações que faremos pro perfil.

**Comment by augustoqm** [comment]
Exato. Então vamos mudar para Perfil dos Vereadores (e ter um filtro para Candidatos desse ano talvez).
Quanto ao último checkbox, já tenho os dados são só esses:

UF  Município    Limite de Gasto Prefeito 1º Turno   Limite de Gasto Prefeito 2º Turno   **Limite de Gasto Vereador**    Limite Contratação Prefeito     **Limite Contratação Vereador**     Eleitorado Apto 
PB  CAMPINA GRANDE   R$ 3,495,432.44     R$ 1,048,629.73     **R$ 144,260.40**   544     **213**     274,006 

Podemos somente comparar com os gastos das eleições passadas, que já estão no BD. 
=)
